docs/core/build/index.md

```diff
@@ -20,7 +20,7 @@ This article gives guidance to developers who want to build and distribute their
 
 ## Build the CLR from source
 
-The source code for the .NET CoreCLR can be found in [the `dotnet/coreclr` repository on GitHub](https://github.com/dotnet/coreclr/).
+The source code for the .NET CoreCLR can be found in [the CoreCLR repository on GitHub](https://github.com/dotnet/coreclr/).
```

Comment:
When we refer to repo names, we usually refer to them as the GitHub path. The sentence as it was tells users *what* it is: the .NET CoreCLR and *where* it is: dotnet/coreclr.

So for consistency, I'd keep it like that.

However, I think we can improve the readability of that sentence by making a change like this:
`can be found in the [dotnet\coreclr](https://github.com/dotnet/coreclr/) repository on GitHub.`

Reply:
That makes sense. I wanted to make it consistent with the link further down the page that looks like this:
`at the base of [the CoreCLR repository](https://github.com/dotnet/coreclr/)`

I'll make both of them the same format then: `the [org-name/repo-name] repository`

Reply:
Good catch @Mackiovello! Yes, that would be perfect!